pyk/requirements/base.txt

```diff
@@ -1 +1,3 @@
 graphviz==0.19.1
+tabulate==0.8.6
+types-tabulate==0.8.6
```

Comment:
This is probably a dev dependency.

Reply:
I'm not sure, I need it downstream in KEVM.